widgetssdk/src/main/java/com/glia/widgets/entrywidget/EntryWidgetContract.kt

```diff
@@ -13,9 +13,9 @@ internal interface EntryWidgetContract {
     }
 
     enum class ItemType {
-        CHAT,
-        AUDIO_CALL,
         VIDEO_CALL,
+        AUDIO_CALL,
+        CHAT,
         SECURE_MESSAGE,
         LOADING_STATE,
         EMPTY_STATE,
```

Comment:
Changed the order in which elements will be shown in the list to make it similar to the design. (I sort media elements based on their enum value)